db/migrate/20220615151407_add_not_null_to_local_authorities.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class AddNotNullToLocalAuthorities < ActiveRecord::Migration[6.1]
+  def change
+    change_column_null :local_authorities, :signatory_name, false
+    change_column_null :local_authorities, :signatory_job_title, false
+    change_column_null :local_authorities, :enquiries_paragraph, false
+    change_column_null :local_authorities, :email_address, false
+    change_column_null :local_authorities, :feedback_email, false
```

Comment:
I think these are all set in our staging/prod db but maybe worth double checking otherwise this migration will fail